app/plugin_api_test.go

```diff
@@ -376,45 +359,10 @@ func TestPluginAPILoadPluginConfiguration(t *testing.T) {
 	th.App.UpdateConfig(func(cfg *model.Config) {
 		cfg.PluginSettings.Plugins["testloadpluginconfig"] = pluginJson
 	})
-	setupPluginApiTest(t,
-		`
-		package main
-
-		import (
-			"github.com/mattermost/mattermost-server/plugin"
-			"github.com/mattermost/mattermost-server/model"
-			"fmt"
-		)
-
-		type configuration struct {
-			MyStringSetting string
-			MyIntSetting int
-			MyBoolSetting bool
-		}
-
-		type MyPlugin struct {
-			plugin.MattermostPlugin
-
-			configuration configuration
-		}
-
-		func (p *MyPlugin) OnConfigurationChange() error {
-			if err := p.API.LoadPluginConfiguration(&p.configuration); err != nil {
-				return err
-			}
-
-			return nil
-		}
-
-		func (p *MyPlugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*model.Post, string) {
-			return nil, fmt.Sprintf("%v%v%v", p.configuration.MyStringSetting, p.configuration.MyIntSetting, p.configuration.MyBoolSetting)
-		}
+	pwd, _ := os.Getwd()
```

Comment:
(nit) 1/5 maybe use `fileutils.FindDir("tests")` like in https://github.com/mattermost/mattermost-server/blob/master/api4/file_test.go#L30

Reply:
Done!